use/startMicrosatellite.Rmd

```diff
@@ -89,7 +98,7 @@ Also, you have an idea of potential violations of the dataset to the null Wright
 
 ## What is next?
 
-You may now want to move into looking into population differentiation (See [Calculating genetic differentiation and clustering methods from microsatellites data](DifferentiationMicrosatellites.html))
+You may now want to move on to looking into population differentiation (See [Calculating genetic differentiation and clustering methods from microsatellites data](2015-12-15-microsatellite-differentiation.html))
```

Comment:
Sorry my nitpicking hits you all the time. Shouldn't we just hyperlink "population differentiation", or is the idea that this would become an expanding list in the future? (And even then, _See_ should be lower case, or start a non-parenthesized sentence, right?)